integrations/gitlab/.port/resources/blueprints.json

```diff
@@ -53,5 +53,99 @@
     "calculationProperties": {},
     "aggregationProperties": {},
     "relations": {}
+  },
+  {
+    "identifier": "member",
+    "title": "Member",
+    "icon": "GitLab",
+    "schema": {
+      "properties": {
+        "state": {
+          "title": "State",
+          "type": "string",
+          "icon": "GitLab",
+          "description": "The current state of the GitLab item (e.g., open, closed)."
+        },
```

Comment:
we are talking about member not gitlab item, please make sure its readable and straight forward for the users